packages/config/config/devices/0x0190/adc-t_3000.json

```diff
@@ -0,0 +1,801 @@
+			"minValue": 10,
+			"maxValue": 1440,
+			"defaultValue": 60,
+			"unsigned": true
+		},
+		{
+			"#": "13",
+			"label": "Fan Circulation Duty Cycle",
+			"unit": "%",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 25
+		},
+		{
+			"#": "14",
+			"label": "Fan Purge Time",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 1,
+			"maxValue": 3600,
+			"defaultValue": 60
+		},
+		{
+			"#": "15",
+			"label": "Maximum Heat Setpoint",
+			"unit": "°F",
+			"valueSize": 4,
+			"minValue": 35,
+			"maxValue": 95,
+			"defaultValue": 95
+		},
+		{
+			"#": "16",
+			"label": "Minimum Heat Setpoint",
+			"unit": "°F",
+			"valueSize": 4,
+			"minValue": 35,
+			"maxValue": 95,
+			"defaultValue": 35
+		},
+		{
+			"#": "17",
+			"label": "Maximum Cool Setpoint",
+			"unit": "°F",
+			"valueSize": 4,
+			"minValue": 50,
+			"maxValue": 95,
+			"defaultValue": 95
+		},
+		{
+			"#": "18",
+			"label": "Minimum Cool Setpoint",
+			"unit": "°F",
+			"valueSize": 4,
+			"minValue": 50,
+			"maxValue": 95,
+			"defaultValue": 50
+		},
+		{
+			"#": "19",
+			"label": "Thermostat Lock",
+			"description": "Lock out physical thermostat controls.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Full Lock",
+					"value": 1
+				},
+				{
+					"label": "Partial Lock",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "20",
+			"label": "Compressor Delay",
+			"valueSize": 1,
+			"unit": "minutes",
+			"minValue": 0,
+			"maxValue": 60,
+			"defaultValue": 5
+		},
+		{
+			"#": "23",
+			"label": "Temperature Display Units",
+			"description": "Celsius or Farenheit for temperature display.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Celsius",
+					"value": 0
+				},
+				{
+					"label": "Farenheit",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "24",
+			"label": "HVAC Modes Enabled",
+			"description": "Which heating/cooling modes are available.",
+			"valueSize": 1,
+			"minValue": 3,
+			"maxValue": 31,
+			"defaultValue": 15,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off, Heat",
+					"value": 3
+				},
+				{
+					"label": "Off, Cool",
+					"value": 5
+				},
+				{
+					"label": "Off, Heat, Cool",
+					"value": 7
+				},
+				{
+					"label": "Off, Heat, Cool, Auto",
+					"value": 15
+				},
+				{
+					"label": "Off, Heat, Emergency Heat",
+					"value": 19
+				},
+				{
+					"label": "Off, Heat, Cool, Emergency Heat",
+					"value": 23
+				},
+				{
+					"label": "Off, Heat, Cool, Auto, Emergency Heat",
+					"value": 31
+				}
+			]
+		},
+		{
+			"#": "25",
+			"label": "Configurable Terminal Setting",
+			"description": "Changes control of configurable terminal",
+			"valueSize": 2,
```

Comment:
Is it possible to specify configuration fields where each byte is a separate thing?

This thermostat for example has 2 configurable terminals, Z1 and Z2, and I believe (though it is not specified in the manual) that each byte will configure a different terminal. There are a few other cases like this, for example parameter 30 has a second byte that reports a status (readonly), whereas the first byte is writable.

Reply:
> Is it possible to specify configuration fields where each byte is a separate thing?

Yeah, [partial parameter](https://zwave-js.github.io/node-zwave-js/#/config-files/file-format?id=partial-parameters). There's also a [bitmask calculator](https://zwave-js.github.io/node-zwave-js/#/config-files/file-format?id=bitmask-calculator) to help you figure them out. And a [guide with examples](https://zwave-js.github.io/node-zwave-js/#/config-files/partial-parameters).